fluster/decoder.py

```diff
@@ -18,8 +18,8 @@
 from abc import ABC, abstractmethod
 from functools import lru_cache
 from shutil import which
-from typing import Type
-
+from typing import List, Type
+import typing
```

Comment:
```suggestion
from typing import List, Type, Any
```

Reply:
Any typing no needed as we'll use [Decoder] instead